tensorboard/components/tf_ng_tensorboard/angular/BUILD

```diff
@@ -0,0 +1,62 @@
+package(default_visibility = ["//tensorboard:internal"])
+
+load("//tensorboard/defs:defs.bzl", "tf_ts_library")
+
+licenses(["notice"])  # Apache 2.0
+
+# This is a dummy rule used as a @angular/common/http dependency.
+# This is not a replacement for @angular/common dependency.
+tf_ts_library(
+    name = "expect_angular_common_http",
+    srcs = [],
+)
+
+# This is a dummy rule used as a @angular/common/testing dependency.
+# This is not a replacement for @angular/common dependency.
+tf_ts_library(
+    name = "expect_angular_common_http_testing",
+    srcs = [],
```

Comment:
Might be a dumb question: should  this declare a dependency on something, like an @npm package?

Reply:
I worry that this statement will be used as a replacement for correct dependency. `@angular/common` should be separately depended on when you are using, for instance, `import {CommonModule} from '@angular/common'`. I can be convinced otherwise.